src/Util/Common.php

```diff
@@ -29,6 +29,23 @@ class Common
         'callable',
     ];
 
+    /**
+     * An array of short variable types for param/var we will check.
+     *
+     * @var string[]
+     */
+    public static $allowedShortTypes = [
+        'array',
+        'bool',
+        'float',
+        'int',
+        'mixed',
+        'object',
+        'string',
+        'resource',
+        'callable',
+    ];
+
```

Comment:
Remove this empty line.

```suggestion
```

Reply:
Please don't request changes which are in violation with the coding standard used by the library.